framework/src/transfers/MultiAppVariableValueSampleTransfer.C

```diff
@@ -52,6 +52,8 @@ MultiAppVariableValueSampleTransfer::MultiAppVariableValueSampleTransfer(
 void
 MultiAppVariableValueSampleTransfer::initialSetup()
 {
+  MultiAppTransfer::initialSetup();
```

Comment:
is there a way to programmatically force all our users' derived classes to call the base class routine first? like some sort of 'override' keyword but like 'override_but_call_anyway'

Reply:
No there is not

Reply:
another let down from C++ lolol